From 4d71610a9262f78767a1fca4d06ae3000100265e Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Thu, 27 Mar 2014 11:04:07 +0300
Subject: [PATCH 129/213] Version 0.8 Revision 5: Wording changed in a bunch of
 text views in AddCowEvent activity. Servicing bull details now not shown if
 event is AI in AddCowEvent activity

---
 MistroFarmer/src/main/AndroidManifest.xml                |  2 +-
 .../org/cgiar/ilri/mistro/farmer/AddEventActivity.java   | 16 ++++++++++++----
 MistroFarmer/src/main/res/values/strings_en.xml          |  6 ++++--
 MistroFarmer/src/main/res/values/strings_sw.xml          |  6 ++++--
 4 files changed, 21 insertions(+), 9 deletions(-)

diff --git a/MistroFarmer/src/main/AndroidManifest.xml b/MistroFarmer/src/main/AndroidManifest.xml
index f7fadbf..2cbe235 100644
--- a/MistroFarmer/src/main/AndroidManifest.xml
+++ b/MistroFarmer/src/main/AndroidManifest.xml
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.cgiar.ilri.mistro.farmer"
     android:versionCode="2"
-    android:versionName="Version 0.8 Revision 4" >
+    android:versionName="Version 0.8 Revision 5" >
 
     <uses-sdk
         android:minSdkVersion="7"
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
index a52c53e..2bc253b 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
@@ -202,10 +202,15 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 eventTypeSelected();
 
                 this.setTitle(Locale.getStringInLocale("bull_servicing",this));
+                dateTV.setText(Locale.getStringInLocale("date_served", this));
                 eventTypeTV.setVisibility(TextView.GONE);
                 eventTypeS.setVisibility(Spinner.GONE);
                 noOfServicingDaysTV.setVisibility(TextView.GONE);
                 noOfServicingDaysET.setVisibility(EditText.GONE);
+                bullNameTV.setVisibility(TextView.VISIBLE);
+                bullNameACTV.setVisibility(AutoCompleteTextView.VISIBLE);
+                bullETNTV.setVisibility(TextView.VISIBLE);
+                bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);
                 remarksTV.setVisibility(TextView.GONE);
                 remarksET.setVisibility(EditText.GONE);
             }
@@ -221,12 +226,13 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 eventTypeSelected();
 
                 this.setTitle(Locale.getStringInLocale("artificial_inseminamtion",this));
+                dateTV.setText(Locale.getStringInLocale("date_of_insemination", this));
                 eventTypeTV.setVisibility(TextView.GONE);
                 eventTypeS.setVisibility(Spinner.GONE);
-                bullNameTV.setVisibility(TextView.VISIBLE);
-                bullNameACTV.setVisibility(AutoCompleteTextView.VISIBLE);
-                bullETNTV.setVisibility(TextView.VISIBLE);
-                bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);
+                bullNameTV.setVisibility(TextView.GONE);
+                bullNameACTV.setVisibility(AutoCompleteTextView.GONE);
+                bullETNTV.setVisibility(TextView.GONE);
+                bullETNACTV.setVisibility(AutoCompleteTextView.GONE);
                 remarksTV.setVisibility(TextView.GONE);
                 remarksET.setVisibility(EditText.GONE);
             }
@@ -349,6 +355,7 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
             okayB.setText(Locale.getStringInLocale("next",this));
         }
         else if(eventTypesEN[eventTypeS.getSelectedItemPosition()].equals("Artificial Insemination")) {
+            dateTV.setText(Locale.getStringInLocale("date_of_insemination", this));
             strawNumberTV.setVisibility(TextView.VISIBLE);
             strawNumberET.setVisibility(EditText.VISIBLE);
             vetUsedTV.setVisibility(TextView.VISIBLE);
@@ -359,6 +366,7 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
             cowIdentifierTV.setVisibility(TextView.VISIBLE);
         }
         else if(eventTypesEN[eventTypeS.getSelectedItemPosition()].equals("Bull Servicing")) {
+            dateTV.setText(Locale.getStringInLocale("date_served", this));
             bullNameTV.setVisibility(TextView.VISIBLE);
             bullNameACTV.setVisibility(AutoCompleteTextView.VISIBLE);
             bullETNTV.setVisibility(TextView.VISIBLE);
diff --git a/MistroFarmer/src/main/res/values/strings_en.xml b/MistroFarmer/src/main/res/values/strings_en.xml
index 3152896..112c6a6 100644
--- a/MistroFarmer/src/main/res/values/strings_en.xml
+++ b/MistroFarmer/src/main/res/values/strings_en.xml
@@ -76,8 +76,8 @@
     <string name="specify_en">Specify</string>
     <string name="straw_number_en">Straw Number</string>
     <string name="vet_used_en">Vet Used</string>
-    <string name="servicing_bull_name_en">Servicing Bull Name</string>
-    <string name="servicing_bull_ear_tag_number_en">Servicing Bull Ear Tag Number</string>
+    <string name="servicing_bull_name_en">Name of the Bull used</string>
+    <string name="servicing_bull_ear_tag_number_en">Ear tag number of the Bull used</string>
     <string name="embryo_number_en">Embryo Number</string>
     <string name="enable_gps_en">Enable GPS</string>
     <string name="reason_for_enabling_gps_en">You need to enable your GPS before you can continue. Press Okay to enable GPS</string>
@@ -174,6 +174,8 @@
     <string name="milk_data_too_old_en">You cannot enter milk production data that is more than one month old</string>
     <string name="milk_too_much_en">The quantity of milk you entered is too much</string>
     <string name="start_date_en">Start Date</string>
+    <string name="date_served_en">Date Served</string>
+    <string name="date_of_insemination_en">Date of insemination</string>
     <string name="no_of_days_in_servicing_en">No. of Days in Servicing</string>
     <string name="associated_servicing_en">Bull Services</string>
     <string name="country_not_found_en">The country you entered cannot be found</string>
diff --git a/MistroFarmer/src/main/res/values/strings_sw.xml b/MistroFarmer/src/main/res/values/strings_sw.xml
index c59914f..d455e79 100644
--- a/MistroFarmer/src/main/res/values/strings_sw.xml
+++ b/MistroFarmer/src/main/res/values/strings_sw.xml
@@ -76,8 +76,8 @@
     <string name="specify_sw">Bainisha</string>
     <string name="straw_number_sw">Straw Number</string>
     <string name="vet_used_sw">Daktari aliyetumiwa</string>
-    <string name="servicing_bull_name_sw">Servicing Bull Name</string>
-    <string name="servicing_bull_ear_tag_number_sw">Servicing Bull Ear Tag Number</string>
+    <string name="servicing_bull_name_sw">Jina la Ndume aliyetumiwa</string>
+    <string name="servicing_bull_ear_tag_number_sw">Namba ya Sikio ya Ndume aliyetumiwa</string>
     <string name="embryo_number_sw">Embryo Number</string>
     <string name="enable_gps_sw">Wezesha GPS</string>
     <string name="reason_for_enabling_gps_sw">Unahitaji kuwezesha GPS kabla ya kuendelea. Bonyeza \'Sawa\' ili kuwawezesha GPS</string>
@@ -174,6 +174,8 @@
     <string name="milk_data_too_old_sw">haukubaliwi kuripoti tukio iliyo tendeka zaidi ya mwezi mmoja</string>
     <string name="milk_too_much_sw">Kiasi ya maziwa ulioingiza ni mingi sana</string>
     <string name="start_date_sw">Tarehe ya kuanza</string>
+    <string name="date_served_sw">Tarehe ya kukwea</string>
+    <string name="date_of_insemination_sw">Tarehe ya kuingiza</string>
     <string name="no_of_days_in_servicing_sw">Idadi ya masiku katika kitendo ya kukwea</string>
     <string name="associated_servicing_sw">Bull Services</string>
     <string name="country_not_found_sw">Nchi ulioingiza haipatikani</string>
-- 
2.7.1

